src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleCloudStorageDAO.scala

```diff
@@ -22,4 +22,14 @@ trait GoogleCloudStorageDAO {
   def getWorkspaces(userId: String): Seq[WorkspacePermissionsPair]
 
   def getBucketName(workspaceId: String): String
+
```

Comment:
not sure what you're getting at.  do you want a rename to GoogleServicesDAO?  do you want this broken up into two DAOs?  Or what?